.github/workflows/CompatHelper.yml

```diff
@@ -12,5 +12,5 @@ jobs:
       - name: CompatHelper.main()
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          COMPATHELPER_PRIV: ${{ secrets.COMPATHELPER_PRIV }}  # optional
+          COMPATHELPER_PRIV: ${{ secrets.COMPATHELPER_PRIV }}
```

Comment:
A secret `COMPATHELPER_PRIV` and a deploy key should be added to the repo, otherwise CompatHelper won't run any tests: https://github.com/JuliaRegistries/CompatHelper.jl#12-set-up-the-ssh-deploy-key-optional

Reply:
Done.

Reply:
Thank you!